CodeJam.Main/ConnectionStrings/ConnectionStringBase.ConnectionWrapper.cs

```diff
@@ -130,17 +134,19 @@ public bool TryGetStringValue(string keyword, [MaybeNullWhen(false)] out string
 			}
 
 			#region Use only allowed keywords
-			/// <inheritdoc />
+			public override bool TryGetValue(string keyword,
+#if NETCOREAPP30_OR_GREATER
+				[MaybeNullWhen(false)]
+#endif
+				out object value) =>
+				base.TryGetValue(keyword, out value);
+
+			[AllowNull]
 			public override object this[string keyword]
```

Comment:
Is there any reason to write such way and not using `object?` ?

```suggestion
			public override object? this[string keyword]
```

Reply:
The current code follows [base class annotation](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Data.Common/src/System/Data/Common/DbConnectionStringBuilder.cs#L71). Getter never returns null, but setter accepts null values.

Reply:
Sorry, this is confusing, I was reffering to `string? keyword`.
IMO it is much clearer to use `?` instead of attributes.